controllers/topics.controller.js

```diff
@@ -27,9 +27,18 @@ function getApi(req,res,next){
 
 function getArticle(req,res,next){
     const {article_id} = req.params
-    findArticleById(article_id).then((article) => {
-        res.status(200).send({'article':article})
+    if (isNaN(article_id)){
+        res.status(400).send({status: 400, msg: 'Bad Request'}) 
+    }
+    findArticleById(article_id)
+    .then((article) => {
+        if (article.length === 1){
```

Comment:
This is a bit confusing as article is singular but an array (usually plural) and we know it has that article in it. The key should hold an object, not an array, as we are only sending back one thing.
Some tidying up around this from the model onwards would be good.

Reply:
Am still using articles.length logic, but have refactored to only send 1st item in array back. (no need to respond, just keeping track of my refactoring)